src/main/java/com/cdancy/bitbucket/rest/fallbacks/BitbucketFallbacks.java

```diff
@@ -68,6 +71,8 @@
 import com.cdancy.bitbucket.rest.domain.repository.WebHookPage;
 import com.cdancy.bitbucket.rest.domain.sync.SyncState;
 import com.cdancy.bitbucket.rest.domain.sync.SyncStatus;
+import com.cdancy.bitbucket.rest.domain.sshkey.AccessKey;
```

Comment:
:warning: Wrong lexicographical order for 'com.cdancy.bitbucket.rest.domain.sshkey.AccessKey' import. Should be before 'com.cdancy.bitbucket.rest.domain.sync.SyncStatus'. (#QR39:74#)-CHECKSTYLE_CustomImportOrderCheck